src/manipulators/clipHandle.js

```diff
@@ -0,0 +1,25 @@
+import { clipToBox, clip } from '../util/clip';
+
+/**
+ * Clip the handle inside the image or displayed area's boundaries depending on
+ * the defined handle moving options. If both prevention flags are false, it
+ * will not change the handle's position.
+ * @public
+ * @function clipHandle
+ * @memberof Manipulators
+ *
+ * @param {Object} eventData - Data object associated with the event.
+ * @param {Object} handle - The handle to be clipped
+ * @param {Object} options - An object containing the handles' moving options
+ * @returns {void}
+ */
+export default function(eventData, handle, options = {}) {
+  if (options.preventHandleOutsideDisplayedArea) {
+    const { tlhc, brhc } = eventData.viewport.displayedArea;
+
+    handle.x = clip(handle.x, tlhc.x - 1, brhc.x);
```

Comment:
This is because displayedArea was created considering the first pixel as 1 and the last as the size of the image (different from what we are used to with any programming language arrays).
Not totally sure the reasons for this but I think it's because we are dealing with DICOM pixel positioning.